docs/howto/update_major_package_spec.md

```diff
@@ -59,3 +59,30 @@ some of the expected values.
 
 For example if a document contains `event.category: web`, the value of
 `event.type` must be `access`, `error` or `info` according to ECS 8.4.
+
+### field "event.dataset" should have value ..., it has ...
+
+The field `event.dataset` should contain the name of the package and the name of
+the data stream that generates it, separated by a dot. For example for documents
+of the "access" data stream of the Apache module, it should be `apache.access`.
+
+To fix this, review what is generating the value of this field. Some ways to fix
+it are the following.
+
+Fix the value in the field definition using a `constant_keyword`:
+```
+- name: event.dataset
+  type: constant_keyword
```

Comment:
This is not possible now. We don't allow to override the type of an imported value. But I think it would make sense to support the change from keyword to constant_keyword. I will open a PR to support it.

Reply:
Added in https://github.com/elastic/elastic-package/pull/969.